rbac/management/access/view.py

```diff
@@ -161,3 +164,26 @@ def validate_and_get_param(self, params):
         if ordering:
             sub_key = f"{app}&order:{ordering}"
         return sub_key, ordering
+
+    def add_ungrouped_hosts_id(self, response, tenant):
+        """Add ungrouped hosts id to the data."""
+        if not settings.ADD_UNGROUPED_HOSTS_ID:
+            return
+        ungrouped_hosts_id = None
+        queried = False
+        for access in response.data["data"]:
```

Comment:
can we just scope it for `inventory:hosts:XXX` permissions ? I think nulls could only for `inventory:host` permissions.

Reply:
Sure, updated